internal/challenge/implme/basic/arithmetics/arithmetics.go

```diff
@@ -1,23 +1,37 @@
 package arithmetics
 
 import (
+	"sync"
+	"sync/atomic"
 	"time"
 )
 
 func SequentialSum(inputSize int) int {
 	sum := 0
-	for i := 1; i <= inputSize; i++ {
-		sum += process(i)
+	for i := 1; i <= int(inputSize); i++ {
+		sum += int(process(int64(i)))
 	}
 	return sum
 }
 
 // ParallelSum implement this method.
-func ParallelSum(_ int) int {
-	panic("implement me!")
+func ParallelSum(inputSize int) int {
+	var sum atomic.Int64
+	wg := sync.WaitGroup{}
+	wg.Add(inputSize)
+	for i := 1; i <= inputSize; i++ {
+		i := i
+		go func() {
```

Comment:
Please make sure to be context aware and handle errors :) 
I would suggest pick one of the existing design examples we walked through.